datastore/db/models.py

```diff
@@ -133,6 +131,15 @@ def archive_run(self):
     def __str__(self):
         return "%s - %s" % (self.pk, self.datetime)
 
+    def is_in_use(self):
+        """Check if this GetterRun is included in any Latest best."""
+        return GetterRun.all_in_use().filter(pk=self.pk).exists()
+
+    @classmethod
+    def all_in_use(cls):
```

Comment:
Having just written this, I've realised a glaring flaw in the plan - as implemented, `all_in_use()` includes GetterRuns from *any* of the three Latest instances, but for the API I'd guess we only want to include Publishers linked to the the CURRENT Latest instance, so it's not the same.

We could have two different ideas:
* `is_in_use`  - in use by *any* Latest - those we don't want to delete
* `is_current` - in use by CURRENT Latest - those we want to show in the API

Also after discussion with Marion this morning, we've decided not to keep any not-in-use data around, and delete it all, so I'm planning to add an `--all-not-in-use` flag to `delete_datagetter_data`, so having a shortcut is helpful.

(Sidenote: I was originally going to make an `--all` flag as opposed to `--all-not-in-use` and just letting the fallback `--force-delete-in-use-data` take care of not deleting everything that's in-use too, but I'm thinking it'd be a very rare scenario where we really do want to delete *everything* i.e. empty the database, so to avoid accidentally deleting everything just not add that ability to the command. If we do want to delete *everything*, we can use the shell.)

I feel like the idea way to implement this would be shortcuts on the manager like:
```python
# As used by GetterRun.is_in_use(self)
GetterRun.objects.in_use()

# For delete_datagetter_data.py's --all-not-in-use flag
GetterRun.objects.not_in_use()

# For the API
# filter via SourceFile link to Latest.CURRENT
Publisher.objects.current()
```

What do you think?

Reply:
Prob quickest to have a chat about this one. If we don't think we need any historical data we can avoid the need for `.current()` on the publisher model by also fixing #129 . Good to plan for the API changes, though we can always merge this PR and then make the changes in the API PR, in terms of a hierarchy of priorities this functionality takes precedence at the moment.